### PR TITLE
alpha_dropout #21633

### LINKED ISSUE #21633: alpha_dropout

### PR DESCRIPTION
**Pull request by FraBomba6**
Close #21633
Added a tmp version of alpha_dropout as composition of other ivy methods

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by danielmunioz** [comment]
Hey @FraBomba6! Thanks for contributing!

The tests seem to be passing locally, however the [original implementation](https://pytorch.org/docs/stable/generated/torch.nn.AlphaDropout.html) does not make use of the `training` argument, frontend functions should resemble the original functions as much as possible, so could you please that argument?

Other than that the PR looks in good shape! :)

**Comment by FraBomba6** [comment]
@danielmunioz I've implemented the functional version of the function since the ivy path is _ivy.frontends.torch.nn.functional.dropout_functions_ and following [the documentation](https://pytorch.org/docs/stable/generated/torch.nn.functional.alpha_dropout.html#torch.nn.functional.alpha_dropout) this function has the `training` argument.

If I'm missing something I apologize for that and I'm waiting for a feedback

**Comment by FraBomba6** [comment]
@danielmunioz still I was missing an argument check of [the original implementation](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/Dropout.cpp#L66) and so added it

**Comment by danielmunioz** [comment]
Yes you are right! the `training` argument should be present for this implementation

All other arguments look good, and all tests seem to be passing, plus the function looks in good shape overall!

LGTM, Merging now!

Thanks for contributing :)
